hie-plugin-api/Haskell/Ide/Engine/PluginsIdeMonads.hs

```diff
@@ -343,28 +342,20 @@ getDiagnosticProvidersConfig c = Map.fromList [("applyrefact",hlintOn c)
 -- Monads
 -- ---------------------------------------------------------------------
 
--- | IdeM that allows for interaction with the ghc-mod session
-type IdeGhcM = GM.GhcModT IdeM
+-- | IdeM that allows for interaction with the Ghc session
+type IdeGhcM = GhcT IdeM
+
+--instance GM.MonadIO (GhcT IdeM) where
```

Comment:
Delete commented out code.

Reply:
I think this can stay and used again once we only support ghc 8.6 or higher?